README.md

```diff
@@ -506,6 +507,30 @@ compatible with `-pedantic`.
    - If static linking, additionally link with dbghelp (`-ldbghelp`) on windows or lib dl (`-ldl`)
      on linux.
 
+
+### With CMake FetchContent
```

Comment:
Can we make it more clear that there are two ways to do things here: Either the easy cmake way or the manual way. I wouldn't want anyone to get confused and think the cmake setup is required then the make and library path stuff on top.

Reply:
Sounds good to me, I probably ought to clarify that you still need to copy over the header into your project too, so I'll add that there as well 👍

Reply:
Thanks!